src/components/organisms/bucket-add-field/BucketAddField.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { FC, memo, useState } from "react";
+import { createShema } from "./BucketAddFieldSchema";
+import Button from "components/atoms/button/Button";
+import useInputRepresenter from "custom-hooks/useInputRepresenter";
+import Tab from "components/atoms/tab/Tab";
+import FlexElement from "components/atoms/flex-element/FlexElement";
+import FluidContainer from "components/atoms/fluid-container/FluidContainer";
+import Icon from "components/atoms/icon/Icon";
+import styles from "./BucketAddField.module.scss";
+import { TypeInputType } from "../../../custom-hooks/useInputRepresenter";
+import Modal from "components/atoms/modal/Modal";
+import { TypeModal } from "utils/interface";
+
+type TypeBucketAddField = {
+  name: string;
+  type: TypeInputType;
+  isOpen?: boolean;
+  modalProps?: TypeModal;
+};
+
+const BucketAddField: FC<TypeBucketAddField> = ({ name, type, isOpen, modalProps }) => {
+  const isInnerFieldsType = type === "object" || type === "array";
+  const initialTab = isInnerFieldsType ? 0 : 1;
+  const [activeTab, setActiveTab] = useState(initialTab);
+  const [configurationValue, setConfigurationValue] = useState({
+    definePattern: false,
+    primaryField: false,
+    translatable: false,
+    readonly: false,
+    uniqueValues: false,
+    requiredField: false,
+    selectionOptions: false,
+    index: false,
+  });
+  const configurationMapping = {
+    string: createShema.configuration,
+    number: createShema.configuration,
+    date: createShema.configurationType1,
+    color: createShema.configurationType1,
+    multiselect: createShema.configurationType1,
+    object: createShema.configurationType2,
+    storage: createShema.configurationType2,
+    richtext: createShema.configurationType2,
+    textarea: createShema.configurationTextarea,
+    boolean: createShema.configurationBoolean,
+    location: createShema.configurationLocation,
+    array: createShema.configurationArray,
+  };
+
+  const configurationSchema = configurationMapping[type] || {};
+  const schema = createShema[type] || {};
+  const schemaWithDynamicTitle = {
+    ...schema,
+    title: {
+      ...schema.title,
+      title: name || "",
```

Comment:
because now the default is empty string

```suggestion
      title: name,
```

Reply:
Done